kedro/framework/cli/pipeline.py

```diff
@@ -140,10 +140,19 @@ def delete_pipeline(
     pipeline_artifacts = _get_pipeline_artifacts(metadata, pipeline_name=name, env=env)
 
     files_to_delete = [
+        pipeline_artifacts.pipeline_conf / f"{confdir}_{name}.yml"
+        for confdir in ("parameters", "catalog")
+        if (pipeline_artifacts.pipeline_conf / f"{confdir}_{name}.yml").is_file()
+    ]
+
+    # Since we remove nesting in 'parameters' and 'catalog' folders,
+    # we want to save also the old project's structure for the 'pipeline delete' command for backward compatibility
+    files_to_delete += [
         pipeline_artifacts.pipeline_conf / confdir / f"{name}.yml"
         for confdir in ("parameters", "catalog")
         if (pipeline_artifacts.pipeline_conf / confdir / f"{name}.yml").is_file()
     ]
+
```

Comment:
Can it be refactored as looping the two patterns? There are many repeat code here, the only difference is `f"{confdir}_{name}.yml"` and  `confdir / f"{name}.yml"`

Reply:
Thanks for catching that, Nok. Fixed it  👍